cli/cmd/configure.go

```diff
@@ -164,6 +164,11 @@ func connectAPI(cfg config) (string, error) {
 		return "", err
 	}
 
+	if resp.StatusCode != 200 {
+			fmt.Println("Unauthorized Access, check your credentials")
```

Comment:
use `go fmt` for  automatic formatting. the indentation in weird

Reply:
fixed